zenodo/config.py

```diff
@@ -219,6 +221,13 @@ def _(x):
         'task': 'zenodo.modules.utils.tasks.file_integrity_report',
         'schedule': crontab(minute=0, hour=7),  # Every day at 07:00 UTC
     },
+    'datacite-metadata-updater': {
+        'task': 'zenodo.modules.records.schedule_update_datacite_metadata',
+        'schedule': timedelta(hours=1),
+        'kwargs': {
+            'max_count': 1000,
```

Comment:
You can use `'max_count': DATACITE_UPDATING_RATE_PER_HOUR` here to be explicit about the value.